openedx/features/colaraz_features/api/views.py

```diff
@@ -68,10 +68,11 @@ def get(self, request, api_method, format=None):
         }
         """
         elgg_id = request.user.colaraz_profile.elgg_id
-        api_details = getattr(settings, 'COLARAZ_NOTIFICATIONS')
-        method_key = self.API_METHODS.get(api_method, None)
+        api_details = getattr(settings, 'COLARAZ_NOTIFICATIONS', {})
+        is_enabled = api_details.get('ENABLE', False)
+        method_key = self.API_METHODS.get(api_method)
 
-        if api_details and elgg_id and method_key:
+        if api_details and is_enabled and elgg_id and method_key:
```

Comment:
I think `is_enabled` only going to `True` if `api_details` have some value we can remove `api_details` from condition.

```suggestion
        if is_enabled and elgg_id and method_key:
```

Reply:
Updated.